mailpoet/lib/Config/Migrator.php

```diff
@@ -1008,29 +1008,14 @@ private function setDefaultValueForLoadingThirdPartyLibrariesForExistingInstalls
   }
 
   private function migrateSegmentDisplaySettingsOnManageSubscriptionPage(): bool {
-    // skip the migration if the DB version is higher than 3.91.1 or is not set (a new installation)
-    if (version_compare($this->settings->get('db_version', '3.102.0'), '3.101.1', '>')) {
+    // skip the migration if the DB version is higher than 3.102.0 or is not set (a new installation)
+    if (version_compare($this->settings->get('db_version', '3.102.1'), '3.102.0', '>')) {
```

Comment:
After yesterday's emergency release, we need to update this code to
`version_compare($this->settings->get('db_version', '3.102.2'), '3.102.1', '>')`

Reply:
Hmm, but if https://github.com/mailpoet/mailpoet/pull/4467 gets merged first (which I think it should), then this should be rewritten to a migration instead.

Reply:
Just checked with @Aschepikov, and the migrations should be ready on Monday. I'll publish a P2 about how to use them once they are merged. It's super easy. This code will need to be moved to a migration.

Reply:
@triple0t I'll drop you a draft of the P2 in a DM.

Reply:
@JanJakes Thanks for stepping in! I was thinking that the new migrations are related only to the new tables related to automations.
@triple0t  I'll assign the PR back to you since it will require the rebase.